recipes/trf/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: trf
+  version: "4.07b"
+
+source:
+  fn: trf407b
+  url: https://tandem.bu.edu/trf/downloads/trf407b.linux64 # [linux]
+  url: https://tandem.bu.edu/trf/downloads/trf407b.macos64 # [osx]
```

Comment:
Well...I wish I could take the sources! But did not find them :/ https://tandem.bu.edu/trf/downloads/